src/popup/components/Modals/QrCodeReader.vue

```diff
@@ -156,9 +157,9 @@ export default {
         ? new Promise((resolve, reject) => {
           this.setQrScanner(true);
           window.plugins.webviewcolor.change('#00FFFFFF');
-
-          window.QRScanner.scan((error, text) => (!error && text ? resolve(text) : reject(error)));
-          window.QRScanner.show();
+          BarcodeScanner.hideBackground();
+          BarcodeScanner.startScan((error, text) => (!error && text ? resolve(text)
+            : reject(error)));
```

Comment:
Try to use this notation for one liners:
```
(something)
  ? firstResult
  : secondResult;
```
So this becomes:
```suggestion
          BarcodeScanner.startScan((error, text) => (!error && text)
            ? resolve(text)
            : reject(error));
```